Duplicati/Library/Backend/TencentCOS/COSBackend.cs

```diff
@@ -81,6 +82,11 @@ public class CosOptions
             /// A path or subfolder in a bucket
             /// </summary>
             public string Path { get; set; }
+            /// <summary>
+            /// Storage class of the object
+            /// https://cloud.tencent.com/document/product/436/33417
```

Comment:
Thank you @moesoha.  Can we provide a link to an English-language page?  That would make this a little more accessible to our current team of developers.  The same link appears in the `Strings.cs` class as well.

Reply:
since Tencent Cloud provides two languages dividually, I have put both en/zh links on it. `Strings.cs` also updated.